Prevent:
error: #error "Compiler or options invalid for pre-UNIX 03 X/Open applications  and pre-2001 POSIX applications

--- comm-esr31/mozilla/media/libpng/pngpriv.h.orig	2015-06-04 13:00:52.516503541 -0700
+++ comm-esr31/mozilla/media/libpng/pngpriv.h	2015-06-04 13:00:52.515657420 -0700
@@ -39,6 +39,8 @@
  */
 #define _POSIX_SOURCE 1 /* Just the POSIX 1003.1 and C89 APIs */
 
+#define _XOPEN_SOURCE 600
+
 #ifndef PNG_VERSION_INFO_ONLY
 /* Standard library headers not required by png.h: */
 #  include <stdlib.h>
